src/praisonai-code/praisonai_code/cli/session/unified.py

```
@@ -28,8 +28,29 @@
+    """Resolve the canonical sessions directory (e.g. ``~/.praisonai/sessions/``).
+
+    The REPL/TUI store shares the same root every other CLI surface reads and
+    writes (``praisonai run``/``session *``/dashboard), so ``code`` sessions
+    land where the rest of the CLI already looks. Falls back to the legacy
+    ``~/.praison/sessions/`` path only if the paths helper cannot be imported.
+    """
+    try:
+        from praisonai_code.cli.configuration.paths import get_sessions_dir
+
+        return get_sessions_dir()
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)